cinema/management/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        connection = None
+        while not connection:
+            try:
+                connection = connections["default"]
+            except OperationalError:
+                self.stdout.write("Database unavailable")
+
+                time.sleep(1)
```

Comment:
The OperationalError exception is caught but not handled properly. If the database is unavailable, the code will enter an infinite loop. Consider adding a maximum number of retries or a timeout for the database connection.